### PR TITLE
Update All Broken links in Docs and Blogs Page

### LINKED ISSUE #868: Broken links in the Docs and Blogs Page
The following links are to be updated or removed to avoid confusion and 404: 
broken link > Source site
1. https://localhost:5173 > https://developer.tbd.website/docs/web5/build/apps/todo-app-tutorial/
2. existing docs > https://developer.tbd.website/open-source/contributing/
3. Verifiable Credentials JSON Schema > https://developer.tbd.website/blog/ssi-tbd-web5/
4. Governance Forums > https://developer.tbd.website/blog/announcing-the-incubation-program/
5. Contributions > https://developer.tbd.website/blog/web5-roadmap/
6. Forums > https://developer.tbd.website/blog/ssi-tbd-web5/
7.  Forum category > https://developer.tbd.website/blog/ssi-tbd-web5/
8. localhost:3000 > https://developer.tbd.website/docs/ssi/ssi-console/
9. jwt.io > https://developer.tbd.website/docs/ssi/create-credentials/
10. https://www.example.com/.well-known/did.json > https://developer.tbd.website/docs/ssi/create-did/
11. https://example.com/.well-known/did.json > https://developer.tbd.website/docs/ssi/create-did/
12. SSI SDK > https://developer.tbd.website/blog/web5-roadmap/
13. SSI Service > https://developer.tbd.website/blog/web5-roadmap/
14. Decentralized Web nodes > https://developer.tbd.website/blog/web5-roadmap/

Kindly assign me this issue so that i can update the links where necessary and remove links for example cases links.

### PR DESCRIPTION
**Pull request by Smartmind12**
## What type of PR is this? (check all applicable)

- [X] ♻️ Refactor
- [ ] ✨ New Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 👷 Example Application
- [ ] 🧑‍💻 Code Snippet
- [ ] 🎨 Design
- [ ] 📖 Content
- [ ] 🧪 Tests
- [ ] 🔖 Release
- [ ] 🚩 Other

## Description

This PR fixes the broken links in the docs and blogs page. 

## Related Tickets & Documents

Resolves #868 

Resolves #

## Added code snippets?
- [ ] 👍 yes
- [X] 🙅 no, because they aren't needed

## Added tests?

- [ ] 👍 yes
- [X] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 readme
- [ ] 📜 contributing.md
- [ ] 📓 general documentation
- [X] 🙅 no documentation needed

### CODE REVIEW

File: site/blog/2022-07-26-web5-roadmap.md [hunk 1]
```diff
@@ -45,12 +45,12 @@ In Quarter 2 (Q2) of 2022, we publicly announced Web5 as an open source communit
 
 ## Milestone 2
 
-In Q3, we’ll deliver initial implementations of the [SSI SDK](https://developer.tbd.website/projects/ssi-service/readme/), [SSI Service](https://developer.tbd.website/projects/ssi-service/README), and [Decentralized Web Nodes](https://developer.tbd.website/projects/dwn-sdk-js/README).
+In Q3, we’ll deliver initial implementations of the [SSI SDK](https://github.com/TBD54566975/ssi-sdk), [SSI Service](https://github.com/TBD54566975/ssi-service), and [Decentralized Web Nodes](https://github.com/TBD54566975/dwn-sdk-js) .
```
Comment (angiejones) [inline site/blog/2022-07-26-web5-roadmap.md:48]:
```suggestion
In Q3, we’ll deliver initial implementations of the [SSI SDK](https://github.com/TBD54566975/ssi-sdk#readme), [SSI Service](https://github.com/TBD54566975/ssi-service#readme), and [Decentralized Web Nodes](https://github.com/TBD54566975/dwn-sdk-js#readme) .
```

File: site/blog/2022-07-26-web5-roadmap.md [hunk 2]
```diff
@@ -45,12 +45,12 @@ In Quarter 2 (Q2) of 2022, we publicly announced Web5 as an open source communit
...
 
 
 ## Milestone 3
 
-In Q1 of 2023, the [SSI SDK](https://developer.tbd.website/projects/ssi-service/readme/), [SSI Service](https://developer.tbd.website/projects/ssi-service/README), and [Decentralized Web Nodes](https://developer.tbd.website/projects/dwn-sdk-js/README) will be feature complete.
+In Q1 of 2023, the [SSI SDK](https://github.com/TBD54566975/ssi-sdk), [SSI Service](https://github.com/TBD54566975/ssi-service), and [Decentralized Web Nodes](https://github.com/TBD54566975/dwn-sdk-js) will be feature complete.
```
Comment (angiejones) [inline site/blog/2022-07-26-web5-roadmap.md:53]:
```suggestion
In Q1 of 2023, the [SSI SDK](https://github.com/TBD54566975/ssi-sdk#readme), [SSI Service](https://github.com/TBD54566975/ssi-service#readme), and [Decentralized Web Nodes](https://github.com/TBD54566975/dwn-sdk-js#readme) will be feature complete.
```

File: site/src/content/open-source/open-source.js [hunk 3]
```diff
@@ -33,31 +33,31 @@ export const content = {
       type: 'button',
       data: {
         label: 'Contributing Guide',
-        url: '/open-source/contributing',
+        url: 'https://developer.tbd.website/open-source/contributing',
```
Comment (angiejones) [inline site/src/content/open-source/open-source.js:36]:
```suggestion
        url: '/open-source/contributing',
```

Don't want to use absolute URLs for internal pages

File: site/src/content/open-source/open-source.js [hunk 4]
```diff
@@ -33,31 +33,31 @@ export const content = {
...
         isExternalLink: false
       },
     },
     {
       type: 'button',
       data: {
         label: 'Project Governance',
-        url: '/open-source/governance',
+        url: 'https://developer.tbd.website/open-source/governance',
```
Comment (angiejones) [inline site/src/content/open-source/open-source.js:44]:
```suggestion
        url: '/open-source/governance',
```

File: site/src/content/open-source/open-source.js [hunk 5]
```diff
@@ -33,31 +33,31 @@ export const content = {
...
         isExternalLink: false
       },
     },
     {
       type: 'button',
       data: {
         label: 'Code of Conduct',
-        url: '/open-source/code-of-conduct',
+        url: 'https://developer.tbd.website/open-source/code-of-conduct',
```
Comment (angiejones) [inline site/src/content/open-source/open-source.js:52]:
```suggestion
        url: '/open-source/code-of-conduct',
```

File: site/src/content/open-source/open-source.js [hunk 6]
```diff
@@ -33,31 +33,31 @@ export const content = {
...
         isExternalLink: false
       },
     },
     {
       type: 'button',
       data: {
         label: 'Incubation Program',
-        url: '/open-source/incubation',
+        url: 'https://developer.tbd.website/open-source/incubation',
```
Comment (angiejones) [inline site/src/content/open-source/open-source.js:60]:
```suggestion
        url: '/open-source/incubation',
```

File: site/src/pages/community/index.js [hunk 7]
```diff
@@ -207,15 +207,15 @@ function CommunityIndex() {
         <div className="grid grid-cols-1 desktop:grid-cols-2 gap-4 pt-7 pb-20">
           <HeroCard
             heroText="Contributor Guide"
-            buttonUrl="../open-source/contributing"
+            buttonUrl="https://developer.tbd.website/open-source/contributing"
```
Comment (angiejones) [inline site/src/pages/community/index.js:210]:
```suggestion
            buttonUrl="/open-source/contributing"
```

File: site/src/pages/community/index.js [hunk 8]
```diff
@@ -207,15 +207,15 @@ function CommunityIndex() {
...
             buttonText="Learn More"
             bodyText="Looking for ways to contribute to the community?"
             themeColor="purple"
             primary
           />
           <HeroCard
             heroText="Incubation Projects"
-            buttonUrl="../open-source/incubation"
+            buttonUrl="https://developer.tbd.website/open-source/incubation"
```
Comment (angiejones) [inline site/src/pages/community/index.js:218]:
```suggestion
            buttonUrl="/open-source/incubation"
```

**Review by angiejones — CHANGES_REQUESTED** [review]

**Comment by Smartmind12** [comment]
Hey @angiejones I have made the necessary updates so kindly look into it!

**Review by angiejones — APPROVED** [review]

**Comment by angiejones** [comment]
thank you @Smartmind12 🙌🏾

@EbonyLouis hacktoberfest

**Pull request merged**